### ISSUE TITLE
Chat doesn't appear when "dock chat" is selected

### ISSUE THREAD

**Issue by ptkato** [vinszent/gnome-twitch#231]
I think it's still there, but off of the window, because I can see a grey 1px to the right of the window.

Started happening after I updated, due to 7eb89a1, I believe.

**Comment by ptkato** [comment]
Ok, it's appearing now, I don't know why that happened.

**Comment by ptkato** [comment]
I don't know if it's related, but none of my chat messages get through.

**Comment by vinszent** [comment]
I've noticed this on some channels as well. They usually require either a verified email or that you've followed for a certain amount of time. 

![screenshot from 2017-03-06 01-09-18](https://cloud.githubusercontent.com/assets/1663668/23593038/d6a8e3b2-0209-11e7-86cd-5c479cfc9cf8.png)

Are you able to chat on those channels on the Twitch website? I'll be adding notifications in the future to tell you when you need to follow, etc.

As for the original issue, it's a confirmed bug that happens under some specific circumstances. I'll put a minimum width on the chat so it doesn't occur anymore.

**Comment by ptkato** [comment]
Yeah, I can chat through Twitch site. My messages sent from Twitch site appears in GT just fine, so I guess it's a problem with the submiting procedure. The writing box gets clean when hitting enter, by the way.

**Comment by vinszent** [comment]
Can you check if 3d1b51e5081d19108664ee4f8041b41bf92cb403 fixed it by any chance?

**Comment by ptkato** [comment]
Working now, seems that it did the trick.

**Comment by vinszent** [comment]
Awesome! I'll leave this issue open however until I fix the original problem you described.
